nifi-extension-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/HandleHttpRequest.java

```diff
@@ -332,7 +339,8 @@ public class HandleHttpRequest extends AbstractProcessor {
             CONTAINER_QUEUE_SIZE,
             MULTIPART_REQUEST_MAX_SIZE,
             MULTIPART_READ_BUFFER_SIZE,
-            PARAMETERS_TO_ATTRIBUTES
+            PARAMETERS_TO_ATTRIBUTES,
+            REQUEST_HEADER_MAX_SIZE
```

Comment:
For consistency, recommend moving this after the Maximum Threads property.

Reply:
No problem, done!